data/integrations/README.md

```diff
@@ -0,0 +1,132 @@
+output:
+    product_category_id: 166 # Apparel & Accessories (Google)
+
+
+# More complex mapping rules (e.g. Shopify -> Amazon)
+input:
+    product_category_id: 126
+    attributes:
+        - name: target_gender
+          value: Female
+output:
+    amazon_category:
+        - 3888171
+        - 9539904011
+input:
+    product_category_id: 123
+output:
+    product_type:
+        - FineRing
+        - Ring
+
+```
+
+#### Generating mappings from rules
+Mappings are generated from the rules as a part of the build/distribution step:
+- Combinations for all input variations are generated for all product categories and their attributes that appear in the inputs found in the set of rules
+  - As an optimization, combinations including attributes that do not occur in any rules are omitted
+- Builder then passes each generated input through the set of rules, reducing the output of each matching rule into a final output
+  - To combine outputs, we do a union of all attributes
+  - When an attributes is present in both outputs being combined an intersection is done
+
+Putting this all together, let us look at how mappings could be compiled for the `Apparel & Accessories > Clothing > Rings` category in Shopify taxonomy to relevant Amazon categories (using attribute/attribute value names and handles for easier reading):
+
+```yaml
+# Rule 1 - Adult Rings
+input:
+    product_category_id: aa-6-9 # Shopify: Apparel & Accessories > Clothing > Rings
+    attributes:
+        - name: age_group
+          value: Adults
+output:
+    amazon_category: # list of all valid categories for adult rings
+        - 3888171 # Men > Jewelry > Rings
+        - 9539904011 # Men > Jewelry > Wedding Rings
+        - 9539894011 # Women > Jewelry > Women's Wedding & Engagement Rings
+        - 9539902011 # Women's Anniversary Rings
+        - 9539895011 # Women's Bridal Rings
+        - 9539896011 # Women's Engagement Rings
+        #...
+```
+
+```yaml
+# Rule 2 - Men's Rings
+input
+    product_category_id: aa-6-9 # Shopify: Apparel & Accessories > Clothing > Rings
+    attributes:
+    - name: target_gender
+      value: Male
+output
+    amazon_category: # list of all valid categories for male rings
+        - 3888171 # Men > Jewelry > Rings
+        - 9539904011 # Men > Jewelry > Wedding Rings
+        - 3880891 # Boys > Jewelry > Rings
+```
+
+```yaml
+# Rule 3 - Women's Rings
+input
+    product_category_id: aa-6-9 # Shopify: Apparel & Accessories > Clothing > Rings
+    attributes:
+        - name: target_gender
+          value: Female
+output
+    amazon_category: # list of all valid categories for female rings
+        - 9539894011 # Women > Jewelry > Women's Wedding & Engagement Rings
+        - 9539902011 # Women's Anniversary Rings
+        - 9539895011 # Women's Bridal Rings
+        - 9539896011 # Women's Engagement Rings
+        - 3881961 # Girls > Jewelry > Rings
+```
+
+**Final Mapping (JSON output)**
```

Comment:
Given we're showing a tableized view, we can probably snip the `(JSON output)` in the header? Otherwise I think we should show actual JSON here